connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -184,13 +184,12 @@ public boolean isSinkConnector(String connName) {
         return workerConnector.isSinkConnector();
     }
 
-    public Connector getConnector(String connType) {
-        Class<? extends Connector> connectorClass = getConnectorClass(connType);
-        return instantiateConnector(connectorClass);
+    public static Connector createConnector(String connType) {
+        return instantiate(getConnectorClass(connType));
     }
 
     @SuppressWarnings("unchecked")
-    private Class<? extends Connector> getConnectorClass(String connectorAlias) {
+    private static Class<? extends Connector> getConnectorClass(String connectorAlias) {
```

Comment:
Since you've noticed that these don't depend on any instance state, do you think it makes sense to move them to a separate file (e.g. `ConnectorUtils`)? The `instantiate` function below would be another candidate.